app/scripts/models/email-resend.js

```diff
@@ -0,0 +1,48 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// This model limits the number of emails a component can send
+
+'use strict';
+
+define([
+  'backbone',
+  'lib/constants'
+], function (Backbone, Constants) {
+
+  function _shouldResend (tries) {
```

Comment:
Our defacto convention is to only use the `_` prefix on private prototype functions or private values. Since this is a hidden internal function, no need for the `_`.